redact/default_agent_redactions.go

```diff
@@ -0,0 +1,36 @@
+package redact
+
+func GetDefaultAgentRedactions() ([]*Redact, error) {
```

Comment:
We should store this builder in the agent pkg, similar to how product defaults are in the product. `Agent` has to know about `Redact`s to be able to apply and pass them, but `redact` the pkg doesn't have to know about the Agent applies its Redact type. It's unlikely in this case, but in future designs: this would have the potential to run into a circular dependency if a default redact ever needed to grow to access `agent` pkg data. Essentially, the circular dep is already there at an abstract design level - but not yet in compiled types.

Reply:
Thanks for pointing that out; I totally agree. This was the very first thing I implemented, before all the product defaults. I'll make it consistent with those.

Reply:
Just a quick follow up: I'd be happy to store this in a separate source file in the agent package, if it makes more sense to do that. Same for the default product-level redactions?

Not sure if that's cleaner than shoving this function into the already-long agent.go file, especially since this is more data than code.

Reply:
AFAIK there isn't much of any golang-level convention on when to split out files within a package. From a compiler view it all gets concatenated together and the order doesn't matter (just like how you can use a function above where it's defined in sourcecode). If `agent/redact.go` makes more sense for you, go for it!